atomsci/ddm/pipeline/chem_diversity.py

```diff
@@ -294,8 +294,7 @@ def upload_distmatrix_to_DS(
     dist_df = pd.DataFrame(dist_matrix)
     dist_df.index = compound_ids
     dist_df.columns = compound_ids
-    fnm = "distmatrix_nm"
-    filename = fn.replace("nm",feature_type)
-    dist_pkl = dist_df.to_pickle(filepath+filename)
+    filename = fn.replace("nm",feature_type) # fn is not defined anywhere. need to address this
```

Comment:
Could this have been `fnm.replace`? I believe `replace` is an instance function on the `str` class. So maybe this should just be:

```python
filename = f"distmatrix_{featrue_type}"
```